metricbeat/module/couchbase/_meta/configure-node.sh

```diff
@@ -0,0 +1,38 @@
+set -e
+set -m
+
+/entrypoint.sh couchbase-server &
+
+sleep 1
+
+waitForCouchbase() {
+    echo -n "Waiting for Couchbase to start."
+    for ((i=1; i<=90; i++)) do
+        if nc -vz 127.0.0.1 8091 2>/dev/null; then
+            return 0
+        fi
+
+        ((i++))
+        echo -n '.'
+        sleep 1
+    done
+
+    echo
+    echo >&2 "Failed to Start Couchbase"
+}
+
+waitForCouchbase
+
+# Setup index and memory quota
+curl -v -X POST http://127.0.0.1:8091/pools/default -d memoryQuota=300 -d indexMemoryQuota=300
```

Comment:
haha yes, me too, but unfortunately it is not possible by config, i could only set specific initial params via cli instead of rest, but loading a sample database is not possible via cli, as far as i know